docs/source/model_design/designing_vivarium_model/interactive_sim/index.rst

```diff
@@ -233,25 +233,48 @@ these are related to V&V. For more general information and best practices on V&V
     - Example Notebook 
   * - V&V for Risks with Many Categories (E.g., LBWSG)
     - Stratifying simulation outcomes by many categories may be too much of a drain on computation time 
-    - 
+    - `LBWSG exposure <https://github.com/ihmeuw/vivarium_research_ciff_sam/blob/main/model_validation/interactive_simulations/model_5/lbwsg_exposure_model_7.ipynb>`_ 
   * - V&V for Continuous Risks
-    - Summary measures such as mean exposure or proportions below a threshold can be simulation outputs. Interactive sims can verify risk exposure standard deviation, look at spread, or check for outliers. 
-    - 
+    - Summary measures such as mean exposure or proportions below a threshold can be simulation outputs. Interactive sims can verify risk exposure standard deviation, look at spread, or check for outliers. You can also verify correlation between risks. 
+    - Examples: 
+      
+      - `Continous risk standard deviation and visualization <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/interactive_example_continuous_risks.ipynb>`_ 
+      - `Risk correlation <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/interactive_correlation.ipynb>`_ 
   * - V&V for Events with Multiple Risk Factors
     - Stratifying event rates by many risk factors might not be computationally feasible; you can verify risk effects by calculating the event rate at the simulant level.
-    - 
+    - `Multiple risks impacting CVD <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Interactive_RR_GregGraphs.ipynb>`_ 
   * - V&V for Relative Risks based on Continuous Risks 
     - For continuous risks with risk effects, simulant level data is needed to validate risk and outcome rates. 
-    - 
+    - `Same example as prior row <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Interactive_RR_GregGraphs.ipynb>`_ 
   * - Check for Simulant Level Continuity 
     - Can check that simulant values which are not meant to change, remain constant over time (example: propensities)
-    - 
+    - `Testing propensities drift <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Old_VV_unresolved/Interactive_Sim_Tests_06.18.2023_testing_propensity.ipynb>`_
   * - Debugging 
     - This is very general, but simulant level data can be helpful in finding potential issues. Some examples include: propensity drift over time or finding problematic outliers. You can also "remove" parts of the sim to see where a problem might be. 
-    - 
+    - Examples: 
+
+      - `Finding common random number error <https://github.com/ihmeuw/vivarium_research_iv_iron/blob/main/validation/child/interactive_simulations/20221003%20Common%20random%20numbers%20investigation.ipynb>`_. 
```

Comment:
Since we're listing things from different projects, would it be helpful to link to the relevant concept model? I could imagine that being useful context if someone wanted to read these notebooks.

Reply:
Good idea

Reply:
Fair, I added them. I think ideally there would be enough context in the notebooks that you could pull relevant code without reading the concept model. The page just tend to be a bit long and unwieldy!